### ISSUE TITLE
Pages not loading in Connect

### ISSUE THREAD

**Issue by ajefts** [appirio-tech/connect-app#2713]
We experienced this issue during a demo.  The demo was being run from London.  It seemed that the site would partially load but then hang. Here is the error we were able to see in the console.

Mo should be able to help us reproduce it.

![image](https://user-images.githubusercontent.com/3812270/49766934-ad78a080-fca4-11e8-8d61-8b59842ea4e7.png)

**Comment by ajefts** [comment]
@vikasrohit @RishiRajSahu Could take a look at this and see if it indicates a broader issue?

cc @acshields

**Comment by RishiRajSahu** [comment]
@ajefts Seems to transient issue due to network latency. I just tried loading the same URL and loaded well in the first attempt

**Comment by RishiRajSahu** [comment]
<img width="1439" alt="screen shot 2018-12-11 at 10 32 31 am" src="https://user-images.githubusercontent.com/11248478/49779507-528c8b00-fd30-11e8-99db-9e9a7f2820f6.png">

**Comment by RishiRajSahu** [comment]
@vikasrohit could this be because of multiple hops we have, for eg - ALBs and tyk ?

**Comment by vikasrohit** [comment]
@ajefts @RishiRajSahu Yes, it seems to be issue with our network because from our API logs, it says request is fulfilled in less than 100ms. And when I try to connect with the API endpoint using `traceroute` it shows too many hops and sometimes it sticks in AWS infrastructure.

**Comment by ajefts** [comment]
Any ideas on how to tackle this issue?  Maybe @mtwomey has some thoughts on it?

**Comment by vikasrohit** [comment]
@ajefts Yes, we need expert advice from @mtwomey  for this. Meanwhile, we have applied a hotfix on production which now uses gzip compression for a long HTTP request which seems to make visible improvement. I hope, that should reduce the chances of the issue you faced.
